docs/src/architecture/state.md

```diff
@@ -1,74 +1,86 @@
 # State
+The state of the network describes the current condition of all individual accounts and note states. It describes what is currently the case. With its state model, using concurrent offchain state, Polygon Miden aims to realise private transactions, and execution and state bloat minimization. 
```

Comment:
I would make a few changes here, more guidance then a blocker.

The state of the Miden rollup describes the current condition of all accounts and notes. The state model has been designed to enable private transactions and minimise state and execution bloat.

Reply:
Changed it a bit.